client/extensions/zoninator/components/forms/zone-content-form/posts-list.jsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { findIndex, map } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import FormFieldset from 'components/forms/form-fieldset';
+import SortableList from 'components/forms/sortable-list';
+import PostCard from './post-card';
+import RecentPostsDropdown from '../../recent-posts-dropdown';
+import SearchAutocomplete from './../../search-autocomplete';
+
+class PostsList extends Component {
+
+	addPost = ( { push } ) => {
```

Comment:
Ah, a perfect scenario for a curried function: `addPost = ( { push } ) => post => push( post );`